source/dotnet/wholesale-api/Batches/Batches.UnitTests/Application/Batches/StartCalculationHandlerTests.cs

```diff
@@ -49,4 +52,25 @@ public class StartCalculationHandlerTests
             calculationDomainServiceMock.Verify(x => x.StartAsync(batch.Id));
         }
     }
+
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task StartCalculationAsync_LogCalledWithExpectedAttributeName(
```

Comment:
I'm unsure what "attribute" refers to? Perhaps like this?
```suggestion
    public async Task StartCalculationAsync_LogsMessageAsInformation(
```

Reply:
I renamed to '_LogsExpectedMessage'